plugins/topology/README.md

```diff
@@ -81,6 +81,57 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
   ```
 
+- The following must be added in`customResources` component in the [`app-config.yaml`](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) file to view the edit code decorator clicking on which will take you to the Red Hat OpenShift Dev Spaces instance :
+
+  ```yaml
+   kubernetes:
+     ...
+     customResources:
+        - group: 'org.eclipse.che'
+          apiVersion: 'v2'
+          plural: 'checlusters'
+  ```
+
+  Also, ensure that the CheCluster is granted a [`ClusterRole`](https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control). You can use the following code to grant the `ClusterRole` to the CheCluster :
+
+  ```yaml
+    ...
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: backstage-read-only
+    rules:
+      ...
+      - apiGroups:
+          - org.eclipse.che
+        resources:
+          - checlusters
+        verbs:
+          - get
+          - list
```

Comment:
Can you please add this to the Cluster Role as well?

https://github.com/janus-idp/backstage-plugins/blob/main/plugins/topology/manifests/clusterrole.yaml